addons/loyalty/tests/test_loyalty.py

```diff
@@ -184,3 +184,36 @@ def test_prevent_archiving_product_when_archiving_program(self):
         loyalty_program.action_archive()
         # Make sure that the main product didn't get archived
         self.assertTrue(product.active)
+
+    def test_card_description_change(self):
```

Comment:
I think the name is not fitting, it should describe that it's testing the tag name change in rewards.